tools/gen_ort_dockerfile.py

```diff
@@ -32,6 +32,13 @@
 
 FLAGS = None
 
+ORT_TO_TRTPARSER_VERSION_MAP = {
+    '1.9.1': (
+        '21.12',         # Triton container version
+        'release/8.2-EA' # ONNX-Tensorrt parser version
```

Comment:
I think we can wait for the GA version of the parser to merge this?
EA was primarily for us to start testing earlier. 
@kevinch-nv , I assume there will be an 8.2 ga branch created in the next week or two?

Reply:
Right! this PR is meant for us to start testing for 21.12 - Triton-ORT-TRT8.2

Reply:
Yes, the GA branch will be created once the 8.2 GA is pubicaly released.